runtime/router/trie_test.go

```diff
@@ -159,7 +159,7 @@ func TestTriePathsWithPatten(t *testing.T) {
 	trie = NewTrie()
 	tests = []ts{
 		{op: set, path: "/:a", value: "foo"},
-		{op: get, path: "/:a", expectedValue: "foo", expectedParams: []Param{{"a", ":a"}}},
+		{op: get, path: "/:a", expectedValue: "foo", expectedParams: []Param{{"a", "a"}}},
```

Comment:
The param is instead `:a` rather than `a` because the query path is `/:a` (note this is not a pattern, rather a literal path).